Sources/Authentication/AuthenticationHandler.swift

```diff
@@ -174,24 +174,20 @@ extension AuthenticationHandler {
             
```

Comment:
The PR says it's about SwiftLint. What are these things?

Reply:
Is it only  applying SwiftLint to the code?

Reply:
Yeah, fixing warnings from swiftlinting